lib/super_digit.rb

```diff
@@ -1,15 +1,30 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(log10n) where n is num digits in n
+# Space Complexity - O(log10n) where n is num digits in n
 def super_digit(n)
```

Comment:
Notice that the loop goes log10 n times and then you do recursion, so the time complexity will be greater than O(n).